extensions/amp-list/amp-list.md

```diff
@@ -153,11 +153,11 @@ In `<amp-list>`, you can use the [`items`](#items) attribute to render a subset
 
 ### Specifying an overflow
 
-Optionally, the `<amp-list>` component can contain an element with the `overflow` attribute. Add an element with the AMP `overflow` attribute to `amp-list` if the AMP framework is unable to size it as requested. If you include a child element of `amp-list` with the `overflow` attribute, it will appear if one of the following conditions are met:
+Optionally, the `<amp-list>` component can contain an element with the `overflow` attribute. If you include a child element of `amp-list` with the `overflow` attribute, it will appear if all of the following conditions are met:
 
--   The bottom of `amp-list` is below the viewport.
-
--   The bottom of `amp-list` is within 15% of the height of the entire page and within 1000px of the end of the page.
+-   The size specified for the `amp-list` was not large enough to fit its rendered contents.
+-   The bottom of `amp-list` is within viewport.
+-   The bottom of `amp-list` is not within 15% of the height of the entire page and within 1000px of the end of the page.
```

Comment:
I don't understand this really... what does "within 15% of the height of the entire page" mean precisely?

I imagine that "within 1000px of the end of the page" means that it can't be >1000px above the end of the page?

Reply:
Its freakin weird. I didn't know about this either:
https://github.com/ampproject/amphtml/blob/d1b027d1536f150e2eba714b527728b5f2bb844b/src/service/resources-impl.js#L1138-L1145.


Essentially, if the element "is near bottom" of the document then it is eligible for resize. And near bottom is defined as "below the top 85% OR within the last 1000px  of the doc".

Reply:
Can reword to: "The bottom of `amp-list` is not near the bottom of the page - defined as being in the top 85% of the document and not in the bottom 1000px".

🙀

Reply:
Still wordsmithing 🤔

Reply:
giving up